lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -117,14 +120,15 @@ Ufe::SceneItemList UsdHierarchy::createUFEChildList(const UsdPrimSiblingRange& r
     // Return UFE child list from input USD child list.
     Ufe::SceneItemList children;
     for (const auto& child : range) {
-        children.emplace_back(UsdSceneItem::create(fItem->path() + child.GetName(), child));
+        children.emplace_back(
+            UsdSceneItem::create(stripInstanceIndexFromUfePath(path()) + child.GetName(), child));
     }
     return children;
 }
 
 Ufe::SceneItem::Ptr UsdHierarchy::parent() const
 {
-    return UsdSceneItem::create(fItem->path().pop(), prim().GetParent());
+    return UsdSceneItem::create(stripInstanceIndexFromUfePath(path()).pop(), prim().GetParent());
```

Comment:
Not sure I understand this one either.  If you have a point instancer index UFE scene item, by pop'ing its tail, you will still get the parent, no?

And therefore if you do both a stripInstanceIndexFromUfePath() AND a pop, you will have removed one too many path components?

Also, I'm unsure about the call to prim().GetParent().  If I'm a point instancer instance, is that the correct call to make?

Reply:
Good note here. I think you're right that the double-pop here is **not** what we want for point instances, which hopefully jibes with my explanation above.

Reply:
Reverted these changes as part of 95a4d4174531e488facd72c1789ebef3434ff469.